<ui version="4.0" >
 <author>Sascha Peilicke</author>
 <class>KeyPad</class>
 <widget class="QWidget" name="KeyPad" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>KeyPad - Qt virtual keyboard library example</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>4</number>
   </property>
   <property name="margin" >
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string/>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QLineEdit" name="lineEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="keyContainerBox" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QFrame" name="frame" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>4</number>
         </property>
         <item row="0" column="0" >
          <widget class="QVirtualKey" name="key_1" >
           <property name="text" >
            <string>1</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_1</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QVirtualKey" name="key_2" >
           <property name="text" >
            <string>2</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_2</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QVirtualKey" name="key_3" >
           <property name="text" >
            <string>3</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_3</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QVirtualKey" name="key_4" >
           <property name="text" >
            <string>4</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_4</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QVirtualKey" name="key_5" >
           <property name="text" >
            <string>5</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_5</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QVirtualKey" name="key_6" >
           <property name="text" >
            <string>6</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_6</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QVirtualKey" name="key_7" >
           <property name="text" >
            <string>7</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_7</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QVirtualKey" name="key_8" >
           <property name="text" >
            <string>8</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_8</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QVirtualKey" name="key_9" >
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>9</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_9</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QVirtualKey" name="key_0" >
           <property name="text" >
            <string>0</string>
           </property>
           <property name="key" >
            <enum>Qt::Key_0</enum>
           </property>
           <property name="layoutHint" >
            <enum>QVirtualKey::EconomicLayoutHint</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QFrame" name="frame_2" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth" >
         <number>1</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>4</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="labelasdasd" >
           <property name="text" >
            <string>Map:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="layoutCombo" >
           <item>
            <property name="text" >
             <string>Numbers</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Characters</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" colspan="2" >
          <widget class="Line" name="line_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2" >
          <widget class="QPushButton" name="enterButton" >
           <property name="text" >
            <string>Enter</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2" >
          <widget class="QPushButton" name="cancelButton" >
           <property name="text" >
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel" >
     <property name="text" >
      <string>Keyboard layout switching example</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVirtualKey</class>
   <extends>QWidget</extends>
   <header>qtvirtualkeyboard/qvirtualkey.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>268</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel" >
     <x>162</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
